test/scenarios/driver/vagrant/molecule/default/molecule.yml

```diff
@@ -16,6 +16,9 @@ platforms:
       - auto_config: true
         network_name: private_network
         type: dhcp
+      - virtualbox__intnet: test_network
```

Comment:
Why is it called `virtualbox__intnet`?

Reply:
Because it is called so in vagrant: https://www.vagrantup.com/docs/virtualbox/networking.html